deps/csl.mk

```diff
@@ -104,4 +104,12 @@ distclean-csl: clean-csl
 
 else
 $(eval $(call bb-install,csl,CSL,true))
+ifeq ($(OS),WINNT)
+install-csl:
+	mkdir -p $(build_private_libdir)/
+	cp -a $(build_libdir)/gcc/$(BB_TRIPLET)/13/libgcc_s.a $(build_private_libdir)/
+	cp -a $(build_libdir)/gcc/$(BB_TRIPLET)/13/libgcc.a $(build_private_libdir)/
+	cp -a $(build_libdir)/gcc/$(BB_TRIPLET)/13/libmsvcrt.a $(build_private_libdir)/
+	cp -a $(build_libdir)/gcc/$(BB_TRIPLET)/13/libssp.dll.a $(build_private_libdir)/
```

Comment:
We will have to write a custom uninstall rule for this too, which is quite tedious. Is the some way we could instead ensure this is added to the PATH instead?

Reply:
Why PATH?  These files are needed only for static linking, not dynamic linking/loading.

Reply:
We can control the linker search path in https://github.com/JuliaLang/julia/blob/86cbb604f853acef34b84416735e1c2640a45d4e/base/linking.jl#L152-L163 but these fill need to be in a place which is accessible from that point.

Reply:
Looks like we should remove `libdir()` when installed (which is where the system might put things and break our linking step) and add this folder when not installed?

Reply:
I added an uninstall rule, which seemed easy enough?
I removed `libdir` from the linker invocation; AFAICT `shlibdir` should suffice. Should we then also get rid of the `libjulia.dll.a  libjulia-internal.dll.a  libopenlibm.dll.a` archives we're shipping in there, or are they used somehow else?

Reply:
Yeah, I guess that isn't too bad. I was expect it to require a whole new `UNINSTALL_CSL` rule. Though this should go outside of the `endif` so that it is correctly defined even if the user later changes USE_SYSTEM_CSL and rebuilds